libs/transactions/src/index.ts

```diff
@@ -1,11 +1,14 @@
-export * from './lib/transactions-journey.component';
-export * from './lib/directives/transaction-additional-details.directive';
-
 export * from './lib/services/transactions-journey-config.service';
```

Comment:
I do realise it's not a part of your change, but could you factor out exporting via `*`, it's dangerous from public API stability perspective and I don't think it promotes the best practices.